sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/dashboard/Surveillance/ContactsDashboardPage.java

```diff
@@ -1,11 +1,168 @@
+      By.xpath(
+          "//div[contains(text(),'Unspecified VHF')]/parent::div/following-sibling::div[4]/div");
+  public static final By CONTACTS_HUMAN_RABIES_COUNTER =
+      By.xpath("//div[contains(text(),'Human Rabies')]/parent::div/following-sibling::div[4]/div");
+  public static final By CONTACTS_OTHER_EPIDEMIC_DISEASE_COUNTER =
+      By.xpath(
+          "//div[contains(text(),'Other Epidemic Disease')]/parent::div/following-sibling::div[4]/div");
+  public static final By CONTACTS_NOT_YET_DEFINED_COUNTER =
+      By.xpath(
+          "//div[contains(text(),'Not Yet Defined')]/parent::div/following-sibling::div[4]/div");
+  public static final By CONTACTS_PER_CASE_MIN_COUNTER =
+      By.xpath("//div[contains(text(),'Contacts per Case')]/parent::div/following-sibling::div[1]");
+  public static final By CONTACTS_PER_CASE_MAX_COUNTER =
+      By.xpath("//div[contains(text(),'Contacts per Case')]/parent::div/following-sibling::div[2]");
+  public static final By CONTACTS_PER_CASE_AVERAGE_COUNTER =
+      By.xpath("//div[contains(text(),'Contacts per Case')]/parent::div/following-sibling::div[3]");
+  public static final By CONTACTS_IN_QUARANTINE_COUNTER =
+      By.xpath(
+          "//div[contains(text(),'Contacts in Quarantine')]/parent::div/following-sibling::div");
+  public static final By CONTACTS_PLACED_IN_QUARANTINE_COUNTER =
+      By.xpath(
+          "//div[contains(text(),'Contacts placed in Quarantine')]/parent::div/following-sibling::div");
+  public static final By CONTACTS_NEW_CASES_NOT_PREVIOUSLY_KNOWN_TO_BE_CONTACTS_COUNTER =
+      By.xpath(
+          "//div[contains(text(),'New Cases not Previously Known to Be Contacts')]/parent::div/following-sibling::div");
+  public static final By FOLLOWUP_STATUS_CHART_UNDER_FU = By.xpath("//*[text()='Under F/U']");
+  public static final By FOLLOWUP_STATUS_CHART_LOST_TO_FU = By.xpath("//*[text()='Lost To F/U']");
+  public static final By FOLLOWUP_STATUS_CHART_COMPLETED_FU =
+      By.xpath("//*[text()='Completed F/U']");
+  public static final By FOLLOWUP_STATUS_CHART_CANCELED_FU = By.xpath("//*[text()='Canceled F/U']");
+
+  public static final By CHART_CONTEXT_MENU = By.cssSelector("[class='highcharts-button-symbol']");
+  public static final By PRINT_CHART_OPTION = By.xpath("//li[text()='Print chart']");
+  public static final By DOWNLOAD_PNG_IMAGE_OPTION = By.xpath("//li[text()='Download PNG image']");
+  public static final By DOWNLOAD_JPEG_IMAGE_OPTION =
+      By.xpath("//li[text()='Download JPEG image']");
+  public static final By DOWNLOAD_PDF_DOCUMENT_OPTION =
+      By.xpath("//li[text()='Download PDF document']");
+  public static final By DOWNLOAD_SVG_VECTOR_IMAGE_OPTION =
+      By.xpath("//li[text()='Download SVG vector image']");
+  public static final By DOWNLOAD_CSV_OPTION = By.xpath("//li[text()='Download CSV']");
+  public static final By DOWNLOAD_XLS_OPTION = By.xpath("//li[text()='Download XLS']");
```

Comment:
can we change locators created by visible text to xpath / css? this won't allow us to extend them over DE market if needed